### PR TITLE
Add Core models & fix packaging

### PR DESCRIPTION
**Pull request by luca-drf**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.1
+0.0.1
```
Comment (marcoooo) [inline VERSION:1]: Sure we want a three number versioning here?

Comment (luca-drf) [reply VERSION:1]: I think starting with "MAJOR.MINOR.MICRO" and semantic versioning is probably the most common versioning pattern for working across teams. But we can change it when we actually release and after having had some feedback from other collaborators.

**Review by marcoooo — APPROVED** [review]

File: VERSION (hunk 1 above)
Comment (marcoooo) [reply VERSION:1]: :+1:

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+language: python
+os: linux
+
+python:
+  - "3.8"
```
Comment (JAlvarezJarreta) [inline .travis.yml:5]: We said we would request 3.7+ compatibility for public Ensembl repos, shouldn't we test in 3.7 then?

File: src/tests/databases/mock_db/table.sql [hunk 3]
```diff
@@ -1,17 +1,15 @@
 CREATE TABLE `gibberish` (
-    `id`      INT NOT NULL,
-    `grp`     VARCHAR(20) DEFAULT NULL,
-    `value`   INT DEFAULT NULL,
-    PRIMARY KEY (`id`, `grp`),
-    KEY (`id`)
+    `id`    INT NOT NULL,
+    `grp`   VARCHAR(20) DEFAULT NULL,
```
Comment (JAlvarezJarreta) [inline src/tests/databases/mock_db/table.sql:3]: A primary key cannot be `NULL` (I replaced it by `''` in my tests).

File: src/tests/databases/mock_db/table.sql [hunk 4]
```diff
@@ -1,17 +1,15 @@
 CREATE TABLE `gibberish` (
-    `id`      INT NOT NULL,
-    `grp`     VARCHAR(20) DEFAULT NULL,
-    `value`   INT DEFAULT NULL,
-    PRIMARY KEY (`id`, `grp`),
-    KEY (`id`)
+    `id`    INT NOT NULL,
```
Comment (JAlvarezJarreta) [inline src/tests/databases/mock_db/table.sql:2]: Should be `INTEGER`, not `INT` (to meet the same nomenclature as the next table).

File: src/python/ensembl/core/models.py [hunk 5]
```diff
@@ -0,0 +1,2337 @@
+# WIP: This module is not complete nor fully tested and will most likely change its public interface
```
Comment (JAlvarezJarreta) [inline src/python/ensembl/core/models.py:1]: Missing the LICENSE header.

**Pull request merged**